README.md

```diff
@@ -17,6 +17,11 @@ What are constructor functions and the `new` keyword? What is a prototype? Descr
 
 Your Answer:
 ```text
+constructor functions are objects that use the new keyword and prototypes.
+The new keyword creates a new instance of that function.
+A prototype is a new value added to the original constructor function with different methods and functionalities.
+You would use a constructor function over a prototype when you have multiple values instead to be recorded instead of a few.
+
 
```

Comment:
:+1:  Great response. some thing to add, prototypes can have methods that are shared across all instances. Constructor function handle what traits of our objects may be different instance after instance